zokrates_core/src/flatten/mod.rs

```diff
@@ -1847,18 +1874,17 @@ impl<'ast, T: Field> Flattener<'ast, T> {
 
     fn get_bits(
         &mut self,
-        e: &FlatUExpression<T>,
+        e: &FlatExpression<T>,
```

Comment:
Can you explain the reasoning here? Is it that the `bits` in `FlatUExpression` has the same role as the bits cache?

Reply:
I actually reverted this change, I believe I misunderstood this part of the code